### ISSUE TITLE
pretrian model

### ISSUE THREAD

**Issue by qingwuz** [razvancaramalau/Sequential-GCN-for-Active-Learning#3]
I downloaded the code and trained directly. It seems that the accuracy does not reach the level in the paper. Is there a pre-trained model in the training process

**Comment by razvancaramalau** [comment]
Hey, there is no pre-trained model. You should obtain similar results. Try increasing the number of epochs or vary the batch size. If you are still not able to reproduce close results. I'll run from my end to check as well. Thanks!

**Comment by manza-ari** [comment]
Hi, I am also facing the same issue. I increased epochs and tried with various batch sizes even Random method is not giving Test accuracy of more than 48%.
